modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -68,8 +68,7 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::validParams()
   MooseEnum extrapolated_lower_limit_behavior(
       "ERROR WARN IGNORE EXCEPTION DONOTHING USELIMIT EXTRAPOLATE", "EXTRAPOLATE");
   // Forbid extrapolation of on high end of limit
```

Comment:
I wonder if the option for ignore, warn, and donothing should be removed to, and only allow error or exception. I'd update the comment here too.